torchvision/transforms/transforms.py

```diff
@@ -1052,18 +1054,26 @@ def get_params(brightness, contrast, saturation, hue):
         transforms = []
 
         if brightness is not None:
+            if isinstance(brightness, float):
+                ColorJitter._check_input(brightness, 'brightness')
```

Comment:
We are using the returned value of `_check_input`, so this doesn't actually do anything IMO.

Could we instead just do
```python
brightness = ColorJitter._check_input(brightness, 'brightness')
```
without the conditional on `float`?

Reply:
Yeah that definitely makes sense, not sure how I missed that.